content/carribean/1_intro.md

```diff
@@ -0,0 +1,10 @@
+---
+image: carribean/carribean.png
```

Comment:
I think we can probably combine the first and second panel.

Reply:
Good suggestion! They are combined now!